### DISCUSSION TITLE
Accented domain isn't fully qualified domain

### DISCUSSION THREAD

**Discussion by matloughnane** [vercel/community#203] [Help] [answered]
Hi,

I've just completed a small proof of concept project and deployed to Vercel but when I tried to add a domain [gúrú.work] I'm getting the error that it's not a fully qualified domain (see image). Please note I've tried using the recommended www redirect and a straight add, to no avail.

I'm assuming that it's because of the accented ú in the domain as I've never had a problem before. Does anyone know how I could get this resolved?

<img width="816" alt="Screenshot 2022-01-18 at 18 06 17" src="https://user-images.githubusercontent.com/1440617/149993908-e85bd1d5-e060-4bfa-9082-4e833bedfdb4.png">

Any help would be greatly appreciated - cheers.

Matthew

**Comment by Cl3arglass** [comment, marked as answer]
Hey Matthew,

Thank you for reaching out. One work around is to convert your domain name to puny code with this tool https://www.punycoder.com/ and then add the result as your domain name.

**Reply by matloughnane** [reply]
Cheers - I didn't realise that was a possibility - I'll give it a go now.
